drivers/homeless_summary_report/app/models/homeless_summary_report/report.rb

```diff
@@ -541,12 +541,13 @@ def title_for(household_category, demographic_category)
       # Force measure 2 because it has the largest date range, and we only use this to check to see if
       # someone existed in a demographic category, time is irrelevant for these
       measure = 'Measure 2'
-      demographic_scope = report_scope(measure)
 
       @filter = filter
       base_variant = spec[:base_variant]
       extra_filters = base_variant[:extra_filters] || {}
       @filter.update(extra_filters.merge(sub_spec[:extra_filters] || {}))
+      demographic_scope = report_scope(measure)
+
       # demographic_filter is a method known to filter_scopes
       sub_spec[:demographic_filters].each do |demographic_filter|
         demographic_scope = send(demographic_filter, demographic_scope)
```

Comment:
This becomes redundant for any filter that is already applied in `report_scope` during `HudFilterBase.apply`. For example if `filter_for_veteran_status` is one of the `demographic_filter`s for this sub_spec, it would have already been called on line 549, but we call it again here.

It is not redundant for all filters though, for example `filter_for_dv_currently_fleeing`, `filter_for_rrh_move_in`, `filter_for_indefinite_disabilities`, and more.

Another option would be to call all those filters in HudFilterBase.apply, and get rid of these couple lines of code and the redundant filtering. However I think that would become harder to maintain and debug, because we wouldn't have the direct mapping from filter value to filter function ([here](https://github.com/greenriver/hmis-warehouse/blob/e04e4c31b380d624319e9d65604bf39b88c4c660/drivers/homeless_summary_report/app/models/homeless_summary_report/report.rb#L984-L987)), instead you would need to know that whatever is called in `HudFilterBase.apply` needs to match the filter list here. Also, any removals from HudFilterBase.apply could mess up this report.